internal/rpcserver/interface.go

```diff
@@ -393,13 +393,21 @@ type Chain interface {
 	// ticket pool.
 	TicketPoolValue() (dcrutil.Amount, error)
 
-	// TicketsWithAddress returns a slice of ticket hashes that are currently live
-	// corresponding to the given address.
-	TicketsWithAddress(address dcrutil.Address) ([]chainhash.Hash, error)
+	// TicketsWithAddress returns a slice of ticket hashes that are currently
+	// live corresponding to the given address.
+	TicketsWithAddress(address dcrutil.Address, isTreasuryEnabled bool) ([]chainhash.Hash, error)
 
 	// TipGeneration returns the entire generation of blocks stemming from the
 	// parent of the current tip.
 	TipGeneration() ([]chainhash.Hash, error)
+
+	// TreasuryBalance returns the treasury balance at the provided block.
+	TreasuryBalance(hash *string) (string, int64, int64, []int64, error)
```

Comment:
It's the natural type that we send in. Either we convert before or in the function. Since this is only called from a single place that has a string pointer it is what I decided to send in.

Reply:
I changed this now anyway.  It now accepts a `chainhash.Hash` like everything else and parses the input in the RPC handler and returns the proper error code in the case it can't be parsed, unlike the code that this comment was made against.